commands.js

```diff
@@ -557,6 +558,7 @@ exports.commands = {
 			}
 		} else {
 			if (target === 'on' || target === 'true') {
+				if (room.modjoin) return this.errorReply("Modjoin is already turned on in this room.");
```

Comment:
<code>room.modjoin</code> can be truthy if set to a specific rank so you should check <code>=== true</code> here. You could also check for setting to the same rank below.